src/main/java/mate/academy/connection/ConnectionUtils.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.connection;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exception.DataProcessingException;
+
+public class ConnectionUtils {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
```

Comment:
Ensure that the database URL in `DB_URL` is correct and matches the schema you have set up for this task. The current URL points to `jdbc:mysql://localhost:3306/test`, which may not be the intended schema for your application.